ql/src/test/results/clientpositive/llap/cbo_SortUnionTransposeRule.q.out

```diff
@@ -1006,6 +1027,22 @@ STAGE PLANS:
                           output format: org.apache.hadoop.hive.ql.io.HiveSequenceFileOutputFormat
                           serde: org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe
         Reducer 5 
+            Execution mode: vectorized, llap
```

Comment:
This is caused by enabling the schema merge for all the optimizations. Apparently the greedy operator chain matching logic worked slightly better when it was run w/o ts-merge first and only then executed to also consider merging the schema.

In this patch I will only introduce the new optimization and do the generalization in either in the "downstream merge" patch or completely separately; it will worth because it will enable the `RemoveSemiJoin` mode to consider merging the ts schema